Src/Support/GoogleApis.Core/Apis/Http/ConfigurableMessageHandler.cs

```diff
@@ -173,6 +174,19 @@ public void RemoveExecuteInterceptor(IHttpExecuteInterceptor interceptor)
 
         #endregion
 
+        private int _loggingRequestId = 0;
+
+        private ILogger _instanceLogger = Logger;
+
+        /// <summary>
+        /// For testing only

```

Comment:
@jskeet yeah that is kind of why i asked i didn't remember seeing them here before.     @chrisdunelm I normally follow this one https://msdn.microsoft.com/en-us/library/ff926074.aspx 

I was just curious thanks :)